src/transformState.js

```diff
@@ -4,8 +4,41 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
```

Comment:
Please use arrow function instead of regular function for consistency with other functions in the code.